src/views/project-board/project-bcf/ProjectBcf.vue

```diff
@@ -447,7 +453,10 @@ export default {
       open: openSettings
     } = useToggle();
 
-    const importBcfTopics = async files => {
+    const importBcfTopics = async event => {
+      const files = Array.from(event.target.files);
+      if (files.length === 0) return;
```

Comment:
Is it possible to have 0 files from the file input?
On firefox, I can't validate the form without a file

Reply:
I agree, I don't think it is necessary to check files length.